lib/screens/config/v2/schedule.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Screens.Config.V2.Schedule do
+  @moduledoc false
+
+  @type t :: %__MODULE__{
+          start_dt: DateTime.t(),
+          end_dt: DateTime.t()
```

Comment:
I think I stated differently during our last meeting, but I changed my mind. 😁
Could you include that these are nullable in the type def? You can just add `| nil`:
```suggestion
          start_dt: DateTime.t() | nil,
          end_dt: DateTime.t() | nil
```